### DISCUSSION TITLE
Welcome to tp5 Discussions!

### DISCUSSION THREAD

**Discussion by ShadyFox2004** [ShadyFox2004/tp5#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Henri-ba** [comment]
Bio:
     name: Henri
     waifu:________
     what I like:____
     
what I do with this community:
I'm in charge of utilitaireMatrix for now.

**Reply by ShadyFox2004** [reply]
Its not a place for playing.

**Reply by ShadyFox2004** [reply]
And use issues to say that.
> I'm in charge of utilitaireMatrix for now.

**Comment by WildSource** [comment]
I might be dumb or something but the method lireDictionnaire I can't do it and I don't know why ?

**Reply by ShadyFox2004** [reply]
What do you mean you cant ?

**Reply by WildSource** [reply]
I'm trying to debug it and rewrote the code but it still doesn't work so I don't know what to do
